doc/common/glossary.md

```diff
@@ -20,94 +20,136 @@ To help guide the reader, this glossary provides an introduction to the terminol
-- **Kubernetes Cluster**: A set of machines, called nodes and master, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
+- **Container Runtime:** The software that is responsible for running containers.
+>_*Note:*_ as explained in [OCI Glossary](https://github.com/opencontainers/runtime-spec/blob/master/glossary.md) it reads the configuration files for a
+**Container** from a directory structure, uses that information to create a container, launches a process inside the container, and performs other lifecycle actions.
+
+- **Kubernetes Cluster:** A set of machines, called nodes and master, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
 >_*Note:*_ adapted from [Kubernetes Glossary](https://kubernetes.io/docs/reference/glossary/?all=true#term-cluster).
-- **Kubernetes Master**:	The master node(s) manage the worker nodes and the pods in the cluster. Multiple masters are used to provide a cluster with failover and high availability.
-- **Kubernetes Control Plane**:	The container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers.
-- **Kubernetes Node**:	A node is a worker machine in Kubernetes. A worker node may be a **VM** or physical machine, depending on the cluster. It has local daemons or services necessary to run Pods and is managed by the control plane.
-- **Kubernetes Service**: An abstract way to expose an application running on a set of Pods as a network service.
+
+- **Kubernetes Control Plane:**	The container orchestration layer that exposes the API and interfaces to define, deploy, and manage the lifecycle of containers.
+
+- **Kubernetes Master:**	The master node(s) manage the worker nodes and the pods in the cluster. Multiple masters are used to provide a cluster with failover and high availability.
+
+- **Kubernetes Node:**	A node is a worker machine in Kubernetes. A worker node may be a **VM** or physical machine, depending on the cluster. It has local daemons or services necessary to run Pods and is managed by the control plane.
+
+- **Kubernetes Service:** An abstract way to expose an application running on a set of Pods as a network service.
 >_*Note:*_ This definition from [Kubernetes Glossary](https://kubernetes.io/docs/reference/glossary/?all=true#term-service) uses the term "network service" differently than in ETSI NFV.
 
+- **Pod:**	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
```

Comment:
```suggestion
- **Pod:**	The smallest and simplest Kubernetes object. A pod represents a set of running containers on your cluster. A pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
```

Reply:
As "Pod" is defined in the glossary, I would like to keep it capitalized.